SPIE2016-9913-16/9913-16.tex

```diff
@@ -0,0 +1,303 @@
+\author[a]{Tim Jenness}
+\author[b]{James Bosch}
+\author[c]{Russell Owen}
+\author[c]{John Parejko}
+\author[a]{Jonathan Sick}
+\author[b]{John Swinbank}
+\author[b]{Miguel de Val-Borro}
+
+\affil[a]{LSST Project Management Office, Tucson, AZ, U.S.A.}
+\affil[b]{Princeton University, Princeton, NJ, U.S.A.}
+\affil[c]{University of Washington, Seattle, WA, U.S.A}
+
+\authorinfo{Further author information: (Send correspondence to T.J.)\\T.J.: E-mail: tjenness@lsst.org}
+
+% Option to view page numbers
+\pagestyle{empty} % change to \pagestyle{plain} for page numbers
+\setcounter{page}{301} % Set start page numbering at e.g. 301
+
+\begin{document}
+\maketitle
+
+\begin{abstract}
+  The Large Synoptic Survey Telescope (LSST) will be an 8.4\,m optical survey telescope sited in Chile and capable of imaging the entire sky twice a week.
+  The data rate of approximately 15\,TB per night and the requirements to both issue alerts on transient sources within 60 seconds of observing and also to create annual data releases means that automated data management systems and data processing pipelines are a key deliverable of the LSST construction project.
+  The LSST data management software has been in development since 2004 and, like other software developed in that era such as CASA, is based on a C++ core with a Python control layer.
+  The software consists of nearly quarter of a million lines of code covering the system from fundamental WCS and table libraries to pipeline environments and distributed process execution.
+
+  The Astropy project began in 2011 as an attempt to bring together disparate open source Python projects and build a core standard infrastructure that can be used by and built upon by the astronomy community.
+  This project has been phenomenally successful in the years since it has begun and has grown to be the de facto standard for Python software in astronomy.
+  Astropy brings with it considerable expectations from the community on how astronomy Python software should be developed and it is clear that by the time LSST is fully operational in the 2020s many of the prospective users of the LSST software stack will assume that Astropy software will be integrated.
+
+  In this paper we describe the overlap between the LSST science pipeline software and Astropy software and investigate areas where the LSST software provides new functionality.
+  We also discuss the possibilities of re-engineering the LSST science pipeline software to build upon Astropy, including the option of contributing affiliated packages.
+\end{abstract}
+
+% Include a list of keywords after the abstract
+\keywords{Astronomy Software, Python, Code Reuse}
+
+\section{INTRODUCTION}
+\label{sec:intro}  % \label{} allows reference to this section
+
+The Large Synoptic Survey Telescope (LSST)\cite{2008arXiv0805.2366I,2016SPIE-Kahn} will be an 8.4\,m optical survey telescope sited in Chile\cite{2014SPIE.9145E..1AG} and capable of imaging the entire sky twice a week.
```

Comment:
I know. Me too. Should I not mention LSST at all in the intro? (but then I can't cite the standard LSST paper). Am I meant to have two ways of describing LSST just to prevent the grating feeling?